src/ios/CDVSplashScreen.m

```diff
@@ -60,6 +60,30 @@ - (void)observeValueForKeyPath:(NSString*)keyPath ofObject:(id)object change:(NS
     [self updateImage];
 }
 
+- (UIViewController*)getCurrentViewController {
+    // Grab the view controller that is currently presented.
+    UIViewController *currentViewController = [[[UIApplication sharedApplication] delegate] window].rootViewController;
+    UIViewController *keyRootController = [UIApplication sharedApplication].keyWindow.rootViewController;
+    
+    // Now present the alert on the view controller that is currently presenting.
```

Comment:
What does "alert" refer to here?

Reply:
Comment has wrong text, I will remove it.